AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialEditor_Atom_ExpectsTestTimeout.py

```diff
@@ -28,22 +28,27 @@ def MaterialEditor_TestsTimeout_TimeoutFailure():
 
     :return: None
     """
-    import Atom.atom_utils.atom_tools_utils as atom_tools_utils
+    import time
 
+    import Atom.atom_utils.atom_tools_utils as atom_tools_utils
     from editor_python_test_tools.utils import Report, Tracer, TestHelper
 
     with Tracer() as error_tracer:
+        # 1. Create start_time variable for the callback function to compare against.
+        start_time = time.time() + 12
 
-        # 1. Create callback function for timing out the test.
+        # 2. Create callback function for timing out the test.
         def loop_variable():
+            if time.time() > start_time:
+                return False
             return True
```

Comment:
the way your doing this now why not return True `if time.time() > start_time: return True` then your wait_for_condition doesn't have to look at loop_variable() is False. name this function something like delay_test() and then your Report.result gets the following
`atom_tools_utils.wait_for_condition(lambda: delay_test(), 15))`

Reply:
updated to just wait for the condition to be true and be false otherwise